### ISSUE TITLE
Capitalized works are not being recognized

### ISSUE THREAD

**Issue by bilbe** [Hedera-Lang-Learn/hedera#144]
One possibility is to ignore all capitalized words.

See also #49

**Comment by bilbe** [comment]
The simplest quick fix for this would be to do a lowercase compare at the right location... not sure where that is.

**Comment by jtauber** [comment]
yep, going to work on this once we've fixed #136

**Comment by jtauber** [comment]
#136 is fixed so on this now

**Comment by jtauber** [comment]
I'd like to propose we add 

1) a new resolution status for `ignored-proper-noun` which should not count in coverage calculations
2) the ability to mark a capitalized token that is currently `no-lemma` as `ignored-proper-noun`
3) the ability to mark all capitalized tokens in a text that are currently `no-lemma` as `ignored-proper-noun` (reversible on a token-by-token basis)

We might later consider the ability to have a stop list of proper nouns that we can ignore automatically.

**Comment by IvyJL** [comment]
James's suggestions regarding `ignored-proper-noun` sound great.

**Comment by IvyJL** [comment]
So as of 2020-11-13, it's great that capitalized words (that are not proper nouns) are being lemmatized. But there is a slight problem in how those capitalized words are treated when disambiguating. If there are three instances with only one capitalized (Amo, amo, amo), if I resolve the capped one, then there is no option to apply to all and resolve the uncapped ones. 
If I resolve an uncapped one, then apply to all only resolves the other uncapped one.

In other words, "Apply to all" should work for both capped and uncapped.

![Screen Shot 2020-11-13 at 11 25 08 AM](https://user-images.githubusercontent.com/18600875/99096093-e3ff7a00-25a3-11eb-8b0b-247a76a24753.png)

**Comment by bilbe** [comment]
Still an issue!
